tests/integration/testnet-scripts/cause-doublesign.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -eux
+
+# The gaiad binary
+BIN=$1
+
+VAL_ID=$2
+
+# The chain ID
+CHAIN_ID=$3
+
+# chain's IP address prefix; 7.7.7, 7.7.8...
+# see chain config for details 
+CHAIN_PREFIX=${4:-"7.7.7"}
```

Comment:
What's going on here? The ip prefix is supplied by the config. Using "7.7.7" is just a convention. Why is it hardcoded here?

Reply:
Refactored and removed.

Now it's just `CHAIN_PREFIX=$4`